tests/components/zwave_js/test_cover.py

```diff
@@ -53,10 +83,10 @@ async def test_cover(hass, client, window_cover, integration):
         blocking=True,
     )
 
-    assert len(client.async_send_command.call_args_list) == 2
+    assert len(client.async_send_command.call_args_list) == 3
```

Comment:
Please reset the mock in between service calls. That makes it easier to reason about the calls.